package.json

```diff
@@ -111,7 +111,8 @@
     "jest-react-native": "18.0.0",
     "jest-snapshots-svg": "^0.0.14",
     "lint-staged": "^3.4.1",
-    "prettier": "^1.4.2",
+    "prettier": "^1.5.0",
+    "react-dom": "16.0.0-alpha.12",
```

Comment:
this was causing a tonne of warnings on the CLI when I ran storybooks, so I've added it in and updated storybooks too.